src/examples/harness-with-diagnostics/index.ts

```diff
@@ -0,0 +1,93 @@
+import { base, board, BoardRunner, InputValues } from "@google-labs/breadboard";
+import { run, RunConfig } from "@google-labs/breadboard/harness";
+import { merMake } from "../../util/merMake.js";
+import { InputResolveRequest } from "@google-labs/breadboard/remote";
+
+const myBoard = await board<InputValues>((inputs) => {
+	const { partOne, partTwo } = inputs;
+
+	const outputOne = base.output({
+		$id: "outputOne",
+		schema: {
+			type: "object",
+			properties: {
+				outputMessageOne: { type: "string" },
+			},
+		},
+	});
+
+	const outputTwo = base.output({
+		$id: "outputTwo",
+		schema: {
+			type: "object",
+			properties: {
+				outputMessageTwo: { type: "string" },
+			},
+		},
+	});
+
+	partOne.as("outputMessageOne").to(outputOne);
+	partTwo.as("outputMessageTwo").to(outputTwo);
+
+	return outputTwo;
+}).serialize({
+	title: "My Board",
+	description: "A description of my board",
+	url: ".",
+});
+
+const runner = await BoardRunner.fromGraphDescriptor(myBoard);
+
+const runConfig: RunConfig = {
+	url: ".",
+	kits: [],
+	remote: undefined,
+	proxy: undefined,
+	diagnostics: true,
+	runner: runner,
+};
+
+const iterator = run(runConfig);
+
+let result = await iterator.next();
+
+while (!result.done) {
+	console.log("HarnessRunResult event type", result.value.type);
+	console.log("HarnessRunResult data", result.value.data);
+
+	if (result.value.type === "input") {
```

Comment:
I feel like it could be useful to explicitely handle the different event types in some way (even just logging them maybe) just to further demonstrate the harness, but I'm still not too sure. For now I just let it log the event type and data for each event. 

Would be good to get thoughts and/or suggestions on this.